docs/development/Git.md

```
@@ -295,6 +295,55 @@ git bisect bad
+
+## Solution
+
+```
+git fetch upstream
+git checkout branch
+git rebase - i upstream/master
+```
+
+In the editor delete all commits not part of the PR and
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)